pom.xml

```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>scm-api</artifactId>
-      <version>2.5.1</version>
+      <version>2.6.3</version>
```

Comment:
The idea is to remove the version number actually and let it come from the BOM 👍

Reply:
Fixed in bf0fb77